### PR TITLE
Map and markers

### PR DESCRIPTION
**Pull request by LindseyWeiskopf**
Summary: Add map to frontend and create markers for each listing
[webapp](https://step2020-smallcity.appspot.com/)

After the list of businesses is returned, I create a map instance using the Maps API centered around the location of the first listing returned.

When parsing through all of the listings, I have a createMarker function that creates a marker for each listing on the map.

The styling change allows the map corners to be rounded. I changed the map from a div to an id to accommodate the API

Thanks!

### CODE REVIEW

File: src/main/webapp/script.js [hunk 1]
```diff
@@ -129,4 +132,20 @@ function getZipCode() {
   xhttp.send();
 
   setTimeout(fetchList, 4000);
+}
+
+function initMap(mapLocation) {
```
Comment (ethanimooney) [inline src/main/webapp/script.js:137]: Let's move the map functions into their own .js file

Comment (shaquille-hall) [reply src/main/webapp/script.js:137]: Agreed.

File: src/main/webapp/script.js [hunk 2]
```diff
@@ -129,4 +132,20 @@ function getZipCode() {
...
+  let myLatLng = {lat: mapLocation.lat, lng: mapLocation.lng};
+  map = new google.maps.Map(document.getElementById('map'), {
+    zoom: 13,
+    center: myLatLng,
+    mapTypeControl: false
+  });
+}
+
+function createMarker(listing) {
```
Comment (shaquille-hall) [inline src/main/webapp/script.js:146]: There seems to be a discrepancy between the number of marker shown, and the number of results shown. Perhaps these should be the same?

File: src/main/webapp/script.js (hunk 1 above)
Comment (LindseyWeiskopf) [reply src/main/webapp/script.js]: It has been moved!

File: src/main/webapp/script.js (hunk 2 above)
Comment (LindseyWeiskopf) [reply src/main/webapp/script.js]: Now the map should only show the 15 markers that appear as listings.

Comment (LindseyWeiskopf) [reply src/main/webapp/script.js]: [deployed here!](https://step2020-smallcity.appspot.com)

**Pull request merged**